tests/vasp/test_outcar.py

```diff
@@ -32,6 +32,10 @@ def setUpClass(cls):
             filename = posixpath.join(direc, f)
             cls.file_list.append(filename)
 
+    def setUp(self):
```

Comment:
This is pretty neat! Maybe we could move this (in a separate PR) to `pyiron_base._tests` so that all pyiron unittests inherit this

Reply:
This enables `self.assertEqual(np_array1, np.array2)`? Then I really like your suggestion to make it available more broadly!